docs/r2dbc.md

```diff
@@ -268,6 +268,24 @@ The environment's application default principal impersonates
 SERVICE_ACCOUNT_1 which impersonates SERVICE_ACCOUNT_2 which then
 impersonates the TARGET_SERVICE_ACCOUNT.
 
+## SQL Admin API URL
+
+The Java Connector supports setting the SQL Admin API URL with the
+`ADMIN_ROOT_URL` and `ADMIN_SERVICE_PATH` options. The `ADMIN_ROOT_URL`
+option specifies the URL-encoded root URL of the service and the
+`ADMIN_SERVICE_PATH` option specifies the URL-encoded service path of the
+service.
+
```

Comment:
Please add a few more pieces of information here: 
- Explain why someone may use this: "Some applications need to connect to a Google Cloud API other than the GCP public API."
- Copy the detailed rules from the underlying api docs:`https://cloud.google.com/java/docs/reference/google-api-client/latest/com.google.api.client.googleapis.services.AbstractGoogleClient.Builder#com_google_api_client_googleapis_services_AbstractGoogleClient_Builder_setRootUrl_java_lang_String_`
- This is optional, describe the default value. "If this is not set, the connector will use the public Google Cloud API." 

The default values from the [underlying driver class](https://github.com/googleapis/google-api-java-client-services/blob/7346122e3aea1d088018a17830e55839f4e6a8bf/clients/google-api-services-sqladmin/v1beta4/2.0.0/com/google/api/services/sqladmin/SQLAdmin.java#L61C27-L61C27) are 

```
DEFAULT_ROOT_URL = "https://sqladmin.googleapis.com/"
DEFAULT_SERVICE_PATH = ""
```

Also, here are some more realistic values for the example. I think `tasks/v1/` will confuse people.
```
    .option(ADMIN_ROOT_URL, "https://googleapis.example.com/");
    .option(ADMIN_SERVICE_PATH, "sqladmin/")
```

Reply:
Finally, in a separate PR we will need to add this to jdbc.md, after #1605 is merged.

Reply:
Updated **jdbc.md**

Reply:
Nice.